core/src/test/java/org/apache/iceberg/hadoop/TestHadoopCommits.java

```diff
@@ -421,7 +422,7 @@ public void testConcurrentFastAppends(@TempDir File dir) throws Exception {
         TABLES.create(
             SCHEMA,
             SPEC,
-            ImmutableMap.of(COMMIT_NUM_RETRIES, String.valueOf(threadsCount)),
+            ImmutableMap.of(COMMIT_NUM_RETRIES, "10", COMMIT_MIN_RETRY_WAIT_MS, "10"),
```

Comment:
why are we updating the number of retries? Also with Hadoop the other component that might affect longer durations is the lock manager's timeout/interval settings: https://github.com/apache/iceberg/blob/34cd01ba2e057866cdb13db8f9919bc98e11e638/core/src/main/java/org/apache/iceberg/CatalogProperties.java#L138-L151